src/views/virtualmachinesinstance/details/tabs/disks/hooks/useDisksTableColumns.ts

```diff
@@ -0,0 +1,41 @@
+import * as React from 'react';
+
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+
+const useDisksTableColumns = () => {
+  const { t } = useKubevirtTranslation();
+
+  const columns = React.useMemo(
```

Comment:
why do we need a file for columns? and why `useMemo`?
does `t` change that much?

Reply:
t is there beacuse of react warning its not actually needed, its memo because if the consuming component rerenders this shoudlt. memo here isn't a big must but its harmful either.